test/functional/ui/popupmenu_spec.lua

```diff
@@ -1204,10 +1204,10 @@ describe('builtin popupmenu', function()
     feed('<c-p>')
     screen:expect([[
       some long prefix before the text|
-      {1:^~                        }{n: word  }|
-      {1:~                        }{n: choice}|
-      {1:~                        }{s: text  }|
-      {1:~                        }{n: thing }|
+      {1:^~                    }{n: word      }|
+      {1:~                    }{n: choice    }|
+      {1:~                    }{s: text      }|
+      {1:~                    }{n: thing     }|
```

Comment:
why did this change? Did `PUM_DEF_WIDTH` not function in the same manner as 'pumwidth' now does?

Perhaps it's worth adding an extra test with a non-default 'pumwidth'?

Reply:
These change again in later popupmnu.c 8.0.x patches. vim-patch:8.0.1491 added new code to handle non-default 'pumwidth',

Reply:
complete() `startcol`  + `&pumwidth` must be less than the screen width to display the pum at the `startcol`.